libraries/chain/include/eosio/chain/wasm_interface_private.hpp

```diff
@@ -180,7 +162,6 @@ namespace eosio { namespace chain {
             try {
                Serialization::MemoryInputStream stream((const U8*)bytes.data(),
```

Comment:
need to remove all of this section

Reply:
oha, thanks! that's the reason of the test failure.